src/test/java/com/ixortalk/organization/api/domain/UserTestBuilder.java

```diff
@@ -32,6 +32,7 @@
 public class UserTestBuilder extends ReflectionInstanceTestBuilder<User> {
 
     private String login = nextString("testUser");
+    private String inviteLanguage = "nl";
```

Comment:
Our approach in using test builders is providing default values for required fields and leaving optional fields empty, so this one should remain empty by default.